examples/_client.rb.example

```diff
@@ -36,3 +36,5 @@ puts "Connected to OneView appliance at #{@client.url}\n\n"
 # @firmware_bundle_path = 'hp-firmware-a1b08f8a6b-HPGH-1.1.i386.rpm'
 # @sas_interconnect_type = 'Synergy 12Gb SAS Connection Module'
 # @synergy_enclosure_hostname = 'fe80::2:0:9:1%eth2'
+# @enclosure_name = '0000A66101'
+# @interconnect_name = '0000A66101, interconnect 3'
```

Comment:
If the enclosure and interconnect can't be retrieved automatically, I'd say it's fine to keep that in the _client.rb.example file. However, I'm wondering if it can be retrieved automatically though; can we get the whole list of enclosures, and select the first one that will work? I'm not sure exactly what determines whether or not it will work, but I'm assuming we can determine that programatically if we can do it manually.

And I agree that all related variables (including the enclosure ones) should be kept together. Grouping will make it easier to fill out/update.